tests/integrations/utils.py

```diff
@@ -144,3 +148,39 @@ def get_span_from_agent(weblog_request):
                 return span_child
 
         raise ValueError(f"Span is not found for {weblog_request.request.url}")
+
+
+def delete_sqs_queue(queue_name):
+    queue_url = f"https://sqs.us-east-1.amazonaws.com/601427279990/{queue_name}"
+    sqs_client = boto3.client("sqs")
```

Comment:
It rely on a third party service ?

Reply:
I don't understand the question here. This code cleans up AWS resources that were used for this test, in this case AWS SQS queues

Reply:
So it relies on AWS